content/en/blog/2022/sig_docs_spotlight_blog.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: blog
+title: "Spotlight on SIG Docs"
+date: 29-07-2022
```

Comment:
Use an ISO8601 date:
```suggestion
date: 2022-07-29
```

Right now, it would render as:
https://k8s.dev/blog/1/01/01/sig-docs-spotlight-2022/